Sources/IntegerUtilities/SaturatingArithmetic.swift

```diff
@@ -11,7 +11,7 @@
 
 extension FixedWidthInteger {
   @_transparent @usableFromInline
-  var signExtension: Self { self &>> -1 }
+  var sextOrZext: Self { self >> Self.bitWidth }
```

Comment:
👀

Reply:
I replaced it with `signbit`, because it's not exactly a `sext` or `zext` in the LLVM sense anyway. `signbit` isn't perfect either (it's not a bit, it's the signbit replicated across a word, but it's not public API so we can tweak the naming later, too).